operators/olm_v1/index.adoc

```diff
@@ -6,18 +6,30 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release. {product-title} 4.14 introduces components for a next-generation iteration of OLM as a Technology Preview feature, known during this phase as _{olmv1}_. This updated framework evolves many of the concepts that have been part of previous versions of OLM and adds new capabilities.
+Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release. {product-title} 4.16 includes components for a next-generation iteration of OLM as a Technology Preview feature, known during this phase as _{olmv1}_. This updated framework evolves many of the concepts that have been part of previous versions of OLM and adds new capabilities.
 
 :FeatureName: {olmv1}
 include::snippets/technology-preview.adoc[]
 
-During this Technology Preview phase of {olmv1} in {product-title} 4.14, administrators can explore the following features:
+[id="olmv1-highlights_{context}"]
+== Highlights
+
+During this Technology Preview phase of {olmv1} in {product-title} 4.16, administrators can explore the following highlights:
 
 Fully declarative model that supports GitOps workflows::
 {olmv1} simplifies Operator management through two key APIs:
 +
 --
-* A new `Operator` API, provided as `operator.operators.operatorframework.io` by the new Operator Controller component, streamlines management of installed Operators by consolidating user-facing APIs into a single object. This empowers administrators and SREs to automate processes and define desired states by using GitOps principles.
+* A new `ClusterExtension` API, provided as `operator.operators.operatorframework.io` by the new Operator Controller component, streamlines management of installed Operators by consolidating user-facing APIs into a single object. This empowers administrators and SREs to automate processes and define desired states by using GitOps principles.
```

Comment:
Should be `clusterextension.olm.operatorframework.io` instead of `operator.operators.operatorframework.io`.